docs/topics/con_server-sent-events.adoc

```diff
@@ -0,0 +1,16 @@
+[id="server-sent-events_{context}"]
+= Server-sent events
+
+Server-sent events (SSE) is a reactive HTTP-based messaging protocol.
+SSE works by establishing a connection between the event source and the client, and allowing the event source, such as an application server or a messaging broker, to use this connection to push events to the client when they occur.
```

Comment:
This sentence is quite long. How about splitting it into two:
SSE works by establishing a connection between the event source and the client. The event source, such as an application server or a messaging broker, can use this connection to push events to the client.

Reply:
Good point:) I split the sentence as you suggested and added one more line for context. Also, I apparently wrote _updates_ when I meant to write _events_, so I fixed that, too. Thank you:)